### PR TITLE
Update nuget-deps-tree Api

### PR DESCRIPTION
**Pull request by RobiNino**
Depends on the release of nuget-deps-tree after [nuget-deps-tree#5](https://github.com/jfrog/nuget-deps-tree/pull/5).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -161,7 +161,7 @@
     "dependencies": {
         "fs-extra": "~9.0.0",
         "keytar": "~5.6.0",
-        "nuget-deps-tree": "~0.1.0",
+        "nuget-deps-tree": "file:/path/to/new/module",
```
Comment (yahavi) [inline package.json:164]:
```suggestion
        "nuget-deps-tree": "~0.1.0",
```

File: src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts [hunk 2]
```diff
@@ -30,25 +30,20 @@ export class NugetTreeNode extends DependenciesTreeNode {
...
-            let nameVersionTuple: string[] = this.getNameVersionTuple(dependency.dependency.id);
-            let name: string = nameVersionTuple[0];
-            let version: string = nameVersionTuple[1];
-            if (version) {
-                let childDependencies: any = dependency.directDependencies;
-                let generalInfo: GeneralInfo = new GeneralInfo(name, version, '', NugetUtils.PKG_TYPE);
+            let id: string = dependency.id;
+            let version: string = dependency.version;
+            let childDependencies: any = dependency.dependencies;
+            if (id && version && childDependencies) {
```
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts:36]: It looks like we filter out nodes without children
```suggestion
            if (id && version) {
```

File: src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts [hunk 3]
```diff
@@ -30,25 +30,20 @@ export class NugetTreeNode extends DependenciesTreeNode {
         }
         for (let key in dependencies) {
             let dependency: any = dependencies[key];
-            let nameVersionTuple: string[] = this.getNameVersionTuple(dependency.dependency.id);
-            let name: string = nameVersionTuple[0];
-            let version: string = nameVersionTuple[1];
-            if (version) {
-                let childDependencies: any = dependency.directDependencies;
-                let generalInfo: GeneralInfo = new GeneralInfo(name, version, '', NugetUtils.PKG_TYPE);
+            let id: string = dependency.id;
```
Comment (yahavi) [inline src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts:33]: Isn't the id already contain the version?

**Review by yahavi — CHANGES_REQUESTED** [review]

File: src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts (hunk 3 above)
Comment (RobiNino) [reply src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts:33]: The new structure has the id and version in separate fields. See the README in the nuget package

File: src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts (hunk 2 above)
Comment (RobiNino) [reply src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts:36]: We only filter if one of these fields is null. An empty array is still truthy

**Pull request merged**